RadarSDK/Include/Radar.h

```diff
@@ -463,6 +464,13 @@ typedef void (^_Nonnull RadarLogConversionCompletionHandler)(RadarStatus status,
  */
 + (void)trackVerifiedTokenWithCompletionHandler:(RadarTrackTokenCompletionHandler _Nullable)completionHandler NS_SWIFT_NAME(trackVerifiedToken(completionHandler:));
 
+/**
+ Starts tracking the user's location with device integrity information for location verification use cases.
+ 
+ @warning Note that you must configure SSL pinning before calling this method.
+ */
++ (void)startTrackingVerified:(BOOL)token NS_SWIFT_NAME(startTrackingVerified(token:));
```

Comment:
We already have the pattern above elsewhere, but a minor style change I'd advocate for is naming bools in a more bool-y way, e.g. `useToken` or `encoded` instead of `token` (which elsewhere is a `NSString`)

Reply:
I'd rather follow existing patterns. I think this is fine with good docs